libs/designer/src/lib/ui/templates/review/ReviewExisting.tsx

```diff
@@ -28,7 +29,7 @@ export const ReviewExisting = ({ resourceOverrides }: ReviewExistingProps) => {
       <div className="msla-templates-review-block">
         <Text>{resourceOverrides?.workflowName ?? resourceStrings.WORKFLOW_NAME}</Text>
         <Text weight="semibold" className={styles.actionName}>
-          {existingWorkflowName}
+          {existingWorkflowName ?? templateName}
```

Comment:
Can you revert this change,? this is the workflow name and lets pass in the value from caller, my current pr is doing that.. we are not storing those data here because it might cause other cascading changes which is required for a template